content/de/act/honey/index.md

```diff
@@ -224,3 +229,31 @@ window.addEventListener('load', function() {
     });
```

Comment:
Also, can you please change `Honey Gold Kontostand` to `„Honey Gold“-Kontostand` above? This is making me sad. :D